src/js/editing/Editor.js

```diff
@@ -379,12 +379,13 @@ define([
      */
     this.insertImage = function ($editable, sUrl, filename) {
       async.createImage(sUrl, filename).then(function ($image) {
+        beforeCommand($editable);
         $image.css({
           display: '',
           width: Math.min($editable.width(), $image.width())
         });
-        beforeCommand($editable);
         range.create().insertNode($image[0]);
+        range.createFromNode($image[0]).collapse().select();
```

Comment:
This means.

1. select node: `range.createFromNode($image[0])`
2. collapse range to end: `collapse()`
3. update browser selection: `select()`